br-webui/_includes/header.liquid

```diff
@@ -14,6 +15,14 @@
 		function shutdown() {
 			socket.emit('shutdown');
 		}
+		socket.on('companion version', function(data) {
+			document.getElementById('companion version').innerHTML = data;
+			if (document.getElementById('cv') != null) {
```

Comment:
Remove this if statement, this code will be hard to maintain. The variable should be looked up and handled in the same file where it is defined.